Services/ItemService.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Services.DTO;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Services
+{
+    public class ItemService
+    {
+        public int GetAll(int userId)
+        {
+            return userId;
+        }
+
+        public void Save(ItemDTO itemDTO)
+        {
+           // string saveValidText = itemDTO.Text;
```

Comment:
I believe you need to map ItemDTO to DomainModels.Item as indicated in the assignment, Thank you!